src/cli.py

```diff
@@ -18,6 +19,7 @@ async def cli(args, script_root):
     handler = StreamHandler()
     handler.setFormatter(StandardLogFormatter())
     logger.addHandler(handler)
+    current_package = Package.current()
```

Comment:
Nits:
- `Package` in this project may refer to a dependency or to the project itself. Maybe `Package` should be renamed to `Project` because protostar project may not always be intended to be used as a package.
- current method name is a bit confusing to me. I would name it `get_instance()` or `create()`. Usually, methods and functions should start with a verb.